adapters/faraday/lib/opentelemetry/adapters/faraday.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Adapters
+    module Faraday
+      module_function
+
+      TRACER_NAME = 'faraday'
+      TRACER_VERSION = Gem.loaded_specs[TRACER_NAME].version.to_s
```

Comment:
This won't work if Faraday isn't present, which is a use case we're going to want to accommodate.

Reply:
I've changed the line to handle when faraday isn't explicitly loaded already. :heavy_check_mark: